### DISCUSSION TITLE
Welcome to Tutorial-on-CasADi-with-CPP Discussions!

### DISCUSSION THREAD

**Discussion by zehuilu** [zehuilu/Tutorial-on-CasADi-with-CPP#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GhoshRitika** [comment]
Hi I tried to clone and run example_racecar but I keep getting seg fault(core dumped). The 'nlpsol' is causing the segmentation fault and I'm not sure why?

**Reply by zehuilu** [reply]
Hi, I would say the first thing is to test whether IPOPT and CasADi are installed correctly for C++ usage. Could you confirm whether you could successfully compile and run` example_nlp_code_gen.cpp` and `example_use_code_gen.cpp`?

**Comment by MufengX1e** [comment]
Hi, thanks for the tutorial. It's really helpful. Just want to update that Casadi is now using "main" branch instead of "master" branch for source clone. Check https://github.com/casadi/casadi/wiki/InstallationLinux
